imap-codec/Cargo.toml

```diff
@@ -41,27 +28,32 @@ quirk = [
     "quirk_rectify_numbers",
     "quirk_trailing_space"
 ]
-
 # Make `\r` in `\r\n` optional.
 quirk_crlf_relaxed = []
 # Encode `A ID ()` as `A ID NIL`
-# Observed in ...
-# * GMX
```

Comment:
Cleaned up a bit: We document these in modern-email/defects. We could reference it but for now I just removed.